### PR TITLE
WIP demo suggestions

### PR DESCRIPTION
**Pull request by kailash-manasarovar**
@krofax would you make a few suggestions on the dummy text please?

### CODE REVIEW

File: docs/.vuepress/sidebar/en.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const enSidebar = sidebar({
     {
       text: "Getting started",
       link: "/dev/fundamentals",
-      children: ["/dev/fundamentals/rollups.md", "/dev/fundamentals/zkSync.md", "/dev/fundamentals/interacting.md", "/dev/fundamentals/hyperscaling.md"],
+      children: ["/dev/fundamentals/lorem.md", "/dev/fundamentals/rollups.md", "/dev/fundamentals/zkSync.md", "/dev/fundamentals/interacting.md", "/dev/fundamentals/hyperscaling.md"],
```
Comment (krofax) [inline docs/.vuepress/sidebar/en.ts:13]:
```suggestion
      children: ["/dev/fundamentals/lorem.md", "/dev/fundamentals/rollups.md", "/dev/fundamentals/zkSync-basics.md", "/dev/fundamentals/interacting.md", "/dev/fundamentals/hyperscaling.md"],
```

File: docs/dev/fundamentals/lorem.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+## Augue vitae persius nec ad 
+
+Vix eu brute nulla homero. Et convenire interesset vix. Ad eos facer nemore, est novum altera prompta ea, ei dico eirmod tacimates mea. Has quaeque detracto ea. Vim te magna brute veniam, et aliquip accommodare est, usu te omnes virtute aliquando. Ad eam aliquip voluptatibus.
+
+## Te debet congue mel
+
+Sea veniam inimicus delicatissimi te. Vis in iisque alterum expetendis, quo impedit adipisci te. Ne sonet zril signiferumque per, per verear aliquid deterruisset id.
+
+- Bonorum torquatos sententiae vim et, mea omnium labitur assentior id. 
+- Ad liber scripserit philosophia his.
```
Comment (krofax) [inline docs/dev/fundamentals/lorem.md:12]:
```suggestion
- Bonorum torquatos sententiae vim et, mea omnium tabiture assentior id. 
- Ad liber scripserit philosophia his. 
```

File: docs/dev/fundamentals/lorem.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Lorem ipsum
+
+## Augue vitae persius nec ad 
+
+Vix eu brute nulla homero. Et convenire interesset vix. Ad eos facer nemore, est novum altera prompta ea, ei dico eirmod tacimates mea. Has quaeque detracto ea. Vim te magna brute veniam, et aliquip accommodare est, usu te omnes virtute aliquando. Ad eam aliquip voluptatibus.
+
+## Te debet congue mel
+
+Sea veniam inimicus delicatissimi te. Vis in iisque alterum expetendis, quo impedit adipisci te. Ne sonet zril signiferumque per, per verear aliquid deterruisset id.
```
Comment (krofax) [inline docs/dev/fundamentals/lorem.md:9]:
```suggestion
Sea veniam pasmiscusd limitations te. Vis in iisque alterum expetendis, quo impedit adipisci te. Ne sonet zril signiferumque per, per verear aliquid deterruisset id.
```

**Review by krofax — CHANGES_REQUESTED** [review]

**Comment by kailash-manasarovar** [comment]
Closing after demo.

**Pull request closed without merging**